glyphicons-only-bootstrap/README.md

```diff
@@ -0,0 +1,24 @@
+# glyphicons-only-bootstrap
+
+Bootstrap nicely offers a functionality for a user to customize their bootstrap package source ([Official link](http://getbootstrap.com/customize/?id=76290a4e23bf2c3f61a4e17625be4627)). However, even if we customize it, there still is a normalize.css part in it. For some users who don't want use normalize.css out of bootstrap, it could be a annoyting part. Thus, I deleted the normalize.css part and this glyphicons-only-bootstrap contains only pure glyphicon parts.
+
+
+## Installation
+
+```
+// using npm
+npm install glyphicons-only-bootstrap
+
+// using bower
+bower install glyphicons-only-bootstrap
+```
+
+## Implement to your HTML
+
+```
+<link rel='stylesheet' href='bower_components/glyphicons-only-bootstrap/css/bootstrap.min.css' />
+```
+
+## License
+
+All license belongs to Bootstrap and the intention to build this package is not in commercial usage. If there could be any license issue reported, I will purge this from my repository.
```

Comment:
Remove the part about commercial use, and make sure the license info is present wherever required.

Reply:
I still see the commercial usage comment here.